R/MsBackend.R

```diff
@@ -313,6 +317,11 @@
 #'   `dropNaSpectraVariables()` might still show columns containing `NA` values
 #'   for *core* spectra variables.
 #'
+#' - `cbind2()`: allows to appends multiple spectra variables to the backend at
+#'   once. It does so *blindly* and is therefore at the risk of the user. For a
```

Comment:
Maybe describe what you mean with *blindly* as that might be confusing to the user. Maybe rather say that *the `Spectra` and the values for the new spectra variables have to be in a matching order*. Do we actually allow to replace also spectra variables? if not we should also mention that here.

Reply:
is the definition fine for you in Spectra.R ? Or should oi precise it  the same way ?